tests/src/test/java/tachyon/client/FileInStreamIntegrationTest.java

```diff
@@ -56,6 +57,9 @@
   private static CreateFileOptions sWriteUnderStore;
 
   @Rule
+  public Timeout mGlobalTimeout = Timeout.seconds(60);
```

Comment:
this is per individual test, so i figured 60 seconds is enough ... I am happy to bump it up to 120 seconds (which is by the way the default in Go ;-))